src/file_validator.py

```diff
@@ -103,6 +103,7 @@ def validate_size_md5(self):
                 info[SUBFOLDER_FILE_NAME] = file_name.replace('/', '_').replace('\\', '_')
             else:
                 info[SUBFOLDER_FILE_NAME] = file_name
+            self.field_names.append(SUBFOLDER_FILE_NAME) # add subfolder file name to field names
```

Comment:
This is inside for loop on line 98, won't it add multiple internal_file_name columns?